src/component/contents/exploration/DraftEditorDecoratedLeaves.react.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DraftEditorDecoratedLeaves.react
+ * @format
+ * @flow
+ *
+ * This is unstable and not part of the public API and should not be used by
+ * production systems. This file may be update/removed without notice.
+ */
+
+'use strict';
+
+const DraftOffsetKey = require('DraftOffsetKey');
+const React = require('React');
+const UnicodeBidi = require('UnicodeBidi');
+const UnicodeBidiDirection = require('UnicodeBidiDirection');
+
+import type {BlockNodeRecord} from 'BlockNodeRecord';
+import type ContentState from 'ContentState';
+import type {DraftDecoratorType} from 'DraftDecoratorType';
+import type {BidiDirection} from 'UnicodeBidiDirection';
+import type {List} from 'immutable';
+
+type Props = {
+  block: BlockNodeRecord,
+  children: ?Array<React.Node>,
+  contentState: ContentState,
+  decorator: DraftDecoratorType,
+  decoratorKey: string,
+  direction: BidiDirection,
+  text: string,
+  leafSet: List<any>,
+};
+
+class DraftEditorDecoratedLeaves extends React.Component<Props> {
+  render(): React.Node {
+    const {
+      block,
+      children,
+      contentState,
+      decorator,
+      decoratorKey,
+      direction,
+      leafSet,
+      text,
+    } = this.props;
+
+    const blockKey = block.getKey();
+    const leavesForLeafSet = leafSet.get('leaves');
```

Comment:
Perhaps I'm missing something here, but leafSet is a List, which means get takes numbers.  Is `leafSet` typed correctly?

Reply:
Will double check, could be that it's typed incorrectly